shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/distsql/rdl/impl/DropShardingTableRuleBackendHandler.java

```diff
@@ -65,10 +66,12 @@ private ShardingRuleConfiguration check(final String schemaName, final Collectio
         if (!notExistedTableNames.isEmpty()) {
             throw new ShardingTableRuleNotExistedException(notExistedTableNames);
         }
-        Collection<String> inUsedTableNames = tableNames.stream().filter(each -> ProxyContext.getInstance().getMetaData(schemaName).getSchema().containsTable(each)).collect(Collectors.toList());
-        if (!inUsedTableNames.isEmpty()) {
-            throw new TablesInUsedException(inUsedTableNames);
-        }
+        // TODO
+        // 1. get binding tables by getBindingTables()
+        // 2. Check whether the dropped table rules exist in the binding table rule
+        // 3. throw ShardingTableRulesInUsedException if exist
+        // 4. refactor check() method to return void
```

Comment:
The details of TODO-task can be described in its issue, here we can just keep 
// TODO
or
// TODO # ISSUE_NUMBER
or 
// TODO, SOME COMMENTS

Reply:
ok

Reply:
@tristaZero fixed